frontend/src/components/ChatBox/ChatBoxInfoText.css

```diff
@@ -15,17 +23,20 @@
 
 .chat-box-info.defence-triggered-text {
 	color: var(--chat-triggered-text-colour);
-	margin-bottom: 0.1rem;
 }
 
 .chat-box-info.reset-level-text {
 	color: var(--chat-info-text-colour);
 	border-bottom: 1px solid var(--chat-reset-level-line-colour);
 	line-height: 0.1em;
+	padding: 0;
 	margin: 0.6rem 0.1rem;
```

Comment:
More margins! I assume we want different info boxes to be spaced differently, according to their type, but still have a neat focus ring around them. Maybe nothing we can do about this, but I'll check out the branch and take a look.

Reply:
yes, the reset-level-text message is styled different to the other info-message texts, it has a white box in the background to create this fancy design:
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/a37608ea-fe99-49c9-8e3e-83d1840ba77e)

But that means that the padding the other info-messages have doesn't work for this type of message

Reply:
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/3d19b215-0c56-40fc-a9b6-d76ed59b8f48)

Reply:
Yeah thought it'd be something like that! Fine, leave it alone 😉